src/vs/workbench/services/backup/test/backupFileService.test.ts

```diff
@@ -0,0 +1,164 @@
+/*---------------------------------------------------------------------------------------------
```

Comment:
Any reason these are excluded from `gulp tslint`? This file is excluded from the ghook check even after https://github.com/Microsoft/vscode/pull/15259

```ts
const tslintFilter = [
	'src/**/*.ts',
	'extensions/**/*.ts',
	'!**/*.d.ts',
	'!**/typings/**',
	'!src/vs/base/**/*.test.ts',
	'!extensions/typescript/test/colorize-fixtures/**',
	'!extensions/vscode-api-tests/testWorkspace/**',
	'!src/vs/workbench/**/*.test.ts',
	'!extensions/**/*.test.ts'
];
```